todo_app/app.py

```diff
@@ -2,25 +2,66 @@
 import todo_app.data.session_items as session_items
 from datetime import datetime, timedelta
 from todo_app.view_model import ViewModel
+from flask_login import LoginManager,login_required, UserMixin, login_user, current_user
+from oauthlib.oauth2 import WebApplicationClient
+import os 
+import requests 
+from werkzeug.exceptions import Forbidden
 
+class User(UserMixin):
+    def __init__(self, username):
+        self.id = username
+
+    def writer_role(self): 
+        return self.id == "bob"
```

Comment:
Minor, but it could be nice to make this name an environment variable so you can change who has write access per environment